google/cloud/internal/background_threads_impl.h

```diff
@@ -42,14 +43,18 @@ class CustomerSuppliedBackgroundThreads : public BackgroundThreads {
 class AutomaticallyCreatedBackgroundThreads : public BackgroundThreads {
  public:
   AutomaticallyCreatedBackgroundThreads();
+  explicit AutomaticallyCreatedBackgroundThreads(std::size_t thread_count);
   ~AutomaticallyCreatedBackgroundThreads() override;
 
   CompletionQueue cq() const override { return cq_; }
   void Shutdown();
+  std::size_t pool_size() const { return pool_.size(); }
 
  private:
+  struct Normalized {};
```

Comment:
Nit: I think we'd typically use some variant of "tag" for the name of this type.

Reply:
Done, I went with `NormalizeTag` because it is now saying (to me) "please normalize" whereas before it was saying "trust me the value is normalized".